changedetectionio/__init__.py

```diff
@@ -759,18 +909,28 @@ def api_watch_add():
 
         if form.validate():
 
+            # get action parameter (add paused button value is 'add', watch button value is 'watch'
+            action = request.form.get('action').strip()
+            
             url = request.form.get('url').strip()
             if datastore.url_exists(url):
                 flash('The URL {} already exists'.format(url), "error")
                 return redirect(url_for('index'))
 
             # @todo add_watch should throw a custom Exception for validation etc
             new_uuid = datastore.add_watch(url=url, tag=request.form.get('tag').strip())
-            # Straight into the queue.
-            update_q.put(new_uuid)
-
-            flash("Watch added.")
-            return redirect(url_for('index'))
+            
+            if action == 'add' :
+                datastore.data['watching'][new_uuid]['paused'] = True
+                datastore.needs_write = True
+                flash("Watch added in a paused state.")
```

Comment:
ahhh this addresses https://github.com/dgtlmoon/changedetection.io/issues/202 right?

Reply:
Yes. I was unaware of #202, and I thought of using a button instead of a checkbox.  I think the button looks good though, and there's definitely no confusion with it.